Makefile

```diff
@@ -117,33 +126,29 @@ psql-crdb: kubectl-ctx-kind
 ## purge nats app and storage pvcs
 clean-nats:
 	kubectl delete statefulsets.apps nats && kubectl delete pvc nats-js-pvc-nats-0 nats-jwt-pvc-nats-0
-#
+
 ## set kube ctx to kind cluster
 kubectl-ctx-kind:
 	export KUBECONFIG=~/.kube/config_kind
 	kubectl config use-context kind-kind
 
-# https://gist.github.com/prwhite/8168133
-# COLORS
-GREEN  := $(shell tput -Txterm setaf 2)
-YELLOW := $(shell tput -Txterm setaf 3)
-WHITE  := $(shell tput -Txterm setaf 7)
-RESET  := $(shell tput -Txterm sgr0)
+## Change service to local service instead of upstream. DIR is optional, defaults to "../".
+## Example: `make fleet-scheduler-local ../services/fleet-scheduler` will tell sandbox to use ../services/fleet-scheduler instead of the upstream
```

Comment:
I'd prefer this help to be minimal, suggest checking out the notes on using this command instead